### ISSUE TITLE
Calling DarkNet.SetDarkModeAllowedForWindow((Window)window!, false) still makes  the title bar dark

### ISSUE THREAD

**Issue by dfev77** [Aldaviva/DarkNet#2]
Calling `DarkNet.SetDarkModeAllowedForWindow((Window)window!, false)` still makes the title bar dark.
The workaround for seems to be to not call it if you don't need the black theme, but is ugly.

Using .Net Core 6 & Windows 10 20h2

Reproduction code:

```
private void Application_Startup(object sender, StartupEventArgs args)
{
	bool useDarkTheme = false;
	DarkNet.SetDarkModeAllowedForProcess(useDarkTheme);

	var mainView = new MyWindow();
	mainView.SourceInitialized += (window, _) =>
	{
		//if (usedarkTheme) // <- this IF is the workaround for me
		  DarkNet.SetDarkModeAllowedForWindow((Window)window!, useDarkTheme);
	};

	mainView.Show();
}
```

**Comment by Aldaviva** [comment]
Oh weird, that's definitely not intended. I'll try to take a look at this soon and see if I can fix it. Thanks for the report and repro code.

**Comment by Aldaviva** [comment]
I apparently had a major rewrite in progress that I forgot about at some point. The good news is that it already fixed the problem you identified!
The less good news is that my pending changes still need to be cleaned up and refactored before release, and it will introduce some API/packaging changes (trying to organize method names and parameters in a more intuitive way, for example). I'll try to get this done and published. Thanks for being patient.

**Comment by Aldaviva** [comment]
Released [version 2.0.0, available on NuGet Gallery](https://www.nuget.org/packages/DarkNet/). This should fix the issue you identified.

As part of the 2.0.0 major version upgrade, the package name changed to just `DarkNet` (no separate packages for WPF and Windows Forms). Also the API signatures changed. It's not a drop-in replacement. But it should give you the functionality you are looking for.

Hopefully the new version will be easy to integrate. Sorry for the disruption. The design and functionality of the old version weren't as good as it they could have been.
